clang/lib/Driver/ToolChains/SPIRV.cpp

```diff
@@ -95,7 +95,19 @@ void SPIRV::Linker::ConstructJob(Compilation &C, const JobAction &JA,
   CmdArgs.push_back("-o");
   CmdArgs.push_back(Output.getFilename());
 
+  // Use of --sycl-link will call the clang-sycl-link-wrapper instead of
+  // the default linker (spirv-link).
+  if (Args.hasArg(options::OPT_sycl_link))
+    Linker = ToolChain.GetProgramPath("clang-sycl-link-wrapper");
```

Comment:
I'm not sure I'd call this a wrapper since it doesn't really wrap around anything as far as I'm aware, it's more of a utility that creates a SYCL image from some LLVM-IR. Maybe just `linker`?

Reply:
That sounds better. Will add the change. Thanks

Reply:
Made the change in 679436acf28a4da805dfbdf388ca1c9ce692ad4d

Thanks